src/components/chipsInput/index.tsx

```diff
@@ -146,6 +147,24 @@ const ChipsInput = forwardRef((props: ChipsInputProps, refToForward: React.Ref<a
     );
   }, [chips, leadingAccessory, defaultChipProps, removeMarkedChip, markedForRemoval, maxChips]);
 
+  const isRequired = useMemo(() => {
+    if (!validate) {
+      return false;
+    }
+
+    const inputValidators = Array.isArray(validate) ? validate : [validate];
+    return inputValidators.includes('required');
+  }, [validate]);
+
+  const validateChipsInput = useCallback(() => {
+    return !isRequired || (chips?.length ?? 0) > 0;
+  }, [isRequired, chips]);
+
+  useDidUpdate(() => {
+    // @ts-expect-error
+    fieldRef.current?.validate();
```

Comment:
Need to check 'validateOnChange' prop or move it to the TextField's onChangeText